src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import "semantic-ui-css/semantic.min.css";
+
+
```

Comment:
I know you guys didn't code this part, but if you can fix these spaces might be awesome :D

Reply:
Thanks, I fixed that.